assets/scss/base/_icons.scss

```diff
@@ -114,6 +114,29 @@ Styleguide Icons.Exclamation
 }
 
 /*
+Tick
+
+The tick icon is used to highlight actioned content to the user.
+
+Markup:
+<i class="tick"></i>
```

Comment:
Think this should be a span rather than an <i>. Also the example seems purely visual - it would need some way of conveying the same information to a non-sighted user.